QUICKSTART.md

```diff
@@ -0,0 +1,308 @@
+# OpenTelemetry QuickStart
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+    + [Sampler](#sampler)
+    + [Span Processor](#span-processor)
+    + [Exporter](#exporter)
+- [Tracing](#tracing)
+  * [Create basic Span](#create-basic-span)
+  * [Create nested Spans](#create-nested-spans)
+  * [Span Attributes](#span-attributes)
+  * [Create Spans with events](#create-spans-with-events)
+  * [Create Spans with links](#create-spans-with-links)
+  * [Context Propagation](#context-propagation)
+- [Metrics](#metrics)
+
+<!-- tocstop -->
+
+OpenTelemetry can be used to instrument code for collecting distributed traces and recording metrics.
+For more details, check out the [Specification Overview].
+
+In the following examples, we demonstrate how to configure the OpenTelemetry SDK, create spans and record metrics through the OpenTelemetry API.
+
+[Specification Overview]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md
```

Comment:
Not sure we want end users to refer to the actual Specification - maybe put it at the end, in a "Resources" list?

Reply:
I agree that specification should not be part of this document, but this is the overview of the overall project. Maybe linking the openteletry website?

Reply:
I'd be fine with that, yes.